packages/cosmwasm-stargate/src/signingcosmwasmclient.ts

```diff
@@ -103,6 +105,38 @@ export interface InstantiateOptions {
   readonly admin?: string;
 }
 
+/**
+ * The options of an .instantiate() call.
+ * All properties are optional.
+ */
+export interface Instantiate2Options {
```

Comment:
This should be listed in `index.ts`

Reply:
I found that `Instantiate2Options` is exactly same with `InstantiateOptions` after I delete salt and fixMsg.
So I delete `Instantiate2Options`.